pandas/io/parsers.py

```diff
@@ -348,38 +347,47 @@ def _validate_nrows(nrows):
     return nrows
 
 
+_compression_to_extension = {
```

Comment:
Ahh. Okay fixed that but they're are these `pandas.io.common.ParserError`s that are killing me. https://github.com/dhimmel/pandas/commit/005b673eb924c30b84484331e36d23250b76dad9#commitcomment-20080291